webxrlayers-1.bs

```diff
@@ -889,6 +895,7 @@ MUST be multiplied by to yield the |session|'s [=native WebGL framebuffer resolu
 
 To <dfn>determine the layout attribute</dfn> using an {{XRTextureType}} |textureType|, a {{XRWebGLRenderingContext}} |context| and a {{XRLayerInit}} |init|, the user agent MUST run the following steps:
   1. If |context| is not an {{WebGL2RenderingContext}} and |textureType| is {{"texture-array"}}, throw {{TypeError}} and abort these steps.
+  1. If |textureType| is {{"texture-array"}} and not all of the session’s {{XRView|XRViews}} have the same dimensions, throw an {{InvalidStateError}} and abort these steps.
```

Comment:
You want to use https://immersive-web.github.io/webxr/#xrsession-viewer-reference-space-list-of-views , XRViews are per-frame objects